<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-H5MG9OP" failures="1" tests="4" name="WebClientSaasAppTest" time="22.006" errors="0" timestamp="2022-04-24T22:27:08 IST">
  <testcase classname="com.saas.app.testCases.TC_LoginPage" name="valLoginPageTest" time="8.344"/>
  <testcase classname="com.saas.app.testCases.TC_ContactsPage" name="valCreateNewContact" time="5.829"/>
  <testcase classname="com.saas.app.testCases.TC_ContactsPage" name="valCreateNewContact" time="4.461"/>
  <testcase classname="com.saas.app.testCases.TC_LogoutPage" name="logoutPageTest" time="1.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.saas.app.testCases.TC_LogoutPage.logoutPageTest(TC_LogoutPage.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- logoutPageTest -->
  <testcase classname="com.saas.app.pageObjects.BaseClass" name="@AfterSuite shutDownEnvironment" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite shutDownEnvironment -->
  <testcase classname="com.saas.app.pageObjects.BaseClass" name="@BeforeSuite initEnvironment" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initEnvironment -->
  <testcase classname="com.saas.app.pageObjects.BaseClass" name="@BeforeSuite initEnvironment" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite initEnvironment -->
  <testcase classname="com.saas.app.pageObjects.BaseClass" name="@AfterSuite shutDownEnvironment" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite shutDownEnvironment -->
</testsuite> <!-- WebClientSaasAppTest -->
